editoast/src/views/timetable/similar_trains/graph.rs

```diff
@@ -183,7 +183,7 @@ impl MatchingState {
 
         let (trains, current_waypoint, to_skip) = match (
             self.graph
-                .trains_on_path(&self.current_waypoint, &target_waypoint, 2),
+                .trains_on_path(&self.current_waypoint, &target_waypoint, 20),
```

Comment:
This breaks the algorithm. We want to allow **at most** one waypoint to be skipped to atone for data errors or minor paths mismatches. This change allow at most 19.

```suggestion
                .trains_on_path(&self.current_waypoint, &target_waypoint, 2),
```

Reply:
My bad, it was for test that I forgot to remove. Thanks!